src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,84 @@
+$toggle-track-length: rem(46px);
+$toggle-track-height: rem(24px);
+$toggle-thumb-size: rem(20px);
+
+.Toggle {
+  display: flex;
+  align-items: center;
+
+  &.disabled {
+    opacity: 0.8;
```

Comment:
This number feels a bit magical. Should we put it in a variable with a descriptive name?